src/app/components/apps-view/apps-view.component.html

```diff
@@ -20,10 +20,7 @@
   <div class="dropdown-wrapper">
     <label class="dropdown-label">Partition: </label>
     <mat-form-field>
-      <mat-select
-        [(value)]="partitionSelected"
-        (selectionChange)="onPartitionSelectionChanged($event)"
-      >
+      <mat-select [(value)]="partitionSelected" (selectionChange)="onPartitionSelectionChanged($event)">
```

Comment:
Pls use prettier in your IDE so that it honors prettierrc config file and does not do unnecessary file formatting as part of the PR changes. I see lot of file formatting changes in the files.

Reply:
reverted.